### PR TITLE
Fs 1381 summary page links

### PR DESCRIPTION
**Pull request by Adam-W1**
Added correct links to the application summary page

### CODE REVIEW

File: app/templates/application_submitted.html [hunk 1]
```diff
@@ -24,10 +24,10 @@ <h2 class="govuk-heading-m">What happens next</h2>
           </p>
 
           <p class="govuk-body">
-            <a href="#" class="govuk-link">What did you think of this service?</a> (takes 30 seconds)
+            <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=EGg0v32c3kOociSi7zmVqN48ORk8WN5LlJITE3Swt-lURUNCR0dHMjgxWFZOMTMxQzlOTVIxVkQ0Sy4u" class="govuk-link">What did you think of this service?</a> (takes 30 seconds)
           </p>
 
-          <a href="/landing">
```
Comment (gidsg) [inline app/templates/application_submitted.html:30]: Could we use `url_for` helper here?

File: app/templates/application_submitted.html [hunk 2]
```diff
@@ -24,10 +24,10 @@ <h2 class="govuk-heading-m">What happens next</h2>
...
+          <a href="/account">
```
Comment (gidsg) [inline app/templates/application_submitted.html:30]: Could we use url_for helper here?

Comment (Adam-W1) [reply app/templates/application_submitted.html:30]: good shout 👍

[the remark this answers is not in the document]
Comment (Adam-W1) [reply app/templates/application_submitted.html:30]: thanks @rayhan15243 . myself and @adamdavies1  have agreed to implement the change on this PR

**Review by rayhan15243 — APPROVED** [review]

**Review by gidsg — APPROVED** [review]
👍 🔗

**Pull request merged**